rosdep/python.yaml

```
@@ -2668,6 +2668,12 @@ python-mock:
     xenial_python3: [python3-mock]
     yakkety: [python-mock]
     zesty: [python-mock]
+python-monotonic:
+  arch: [python-monotonic]
+  debian: [python-monotonic]
+  fedora: [python-monotonic]
+  gentoo: [dev-python/monotonic]
+  ubuntu: [python-monotonic]
```

Let's add an entry for opensuse too:
```suggestion
  opensuse: [python2-monotonic]
  ubuntu: [python-monotonic]
```